api/models/userDao.js

```diff
@@ -1,5 +1,18 @@
 const dataSource = require('./dataSource')
 
+const getUserByEmail = async (email) => {
+    const result = await dataSource.query(`
+        SELECT
+              id,
+              name,
+              email,
+              password
+         FROM users
+         WHERE email =?`, [email]
+    )
+    return result [0]
```

Comment:
마이너한 수정 사항이지만 배열을 인덱싱하는 부분을 붙여서 작성해주세요
```suggestion
    return result[0]
```